Userland/Libraries/LibJS/Runtime/ErrorTypes.h

```diff
@@ -327,7 +327,8 @@
                                                       "must be called with the RegExp constructor for the this value")                  \
     M(GetLegacyRegExpStaticPropertyThisValueMismatch, "Legacy RegExp static property getter "                                           \
                                                       "must be called with the RegExp constructor for the this value")                  \
-    M(GetLegacyRegExpStaticPropertyValueEmpty, "Legacy RegExp static property getter value is empty")
+    M(GetLegacyRegExpStaticPropertyValueEmpty, "Legacy RegExp static property getter value is empty")                                   \
+    M(InvalidWasmMultiValueResultCount, "Invalid number of return values for multi-value wasm return of {} objects")
```

Comment:
Please don't put LibWeb error messages in LibJS, inline the string for now or start a similar enumeration over there

Reply:
Alright.